src/components/Shop.js

```diff
@@ -0,0 +1,113 @@
+import styled, { css } from 'styled-components/macro'
+import { useState } from 'react'
+import StoryToShop from './StoryToShop'
+import OrderingForm from './OrderingForm'
+
+function Shop() {
+  const [flipToShop, setFlipToShop] = useState(false)
+  function flipToShopButton() {
+    setFlipToShop(!flipToShop)
+  }
+  return (
+    <Container>
+      <FlipCard flipToShop={flipToShop}>
+        <FlipCardFront>
+          <StoryToShop />
+          <ButtonFront onClick={flipToShopButton}>
+            Lakritzel befreien
```

Comment:
I just asked myself if you could just use `setFlipToShop(!flipToShop)` for `onClick` instead of using a new function. If so you would use less code and the readability would be better, I suppose. Not sure about it, but if you like you can test it and tell me if it works. 🤔